MANIFEST.in

```diff
@@ -1 +1,3 @@
-include tree_hugger/core/assets/queries/*.yml
\ No newline at end of file
+include tree_hugger/core/assets/queries/*.yml
```

Comment:
Maybe we should get rid of this line @CDluznie

Reply:
Should we change by `.../parser/python/queries.yml` and same for others default parsers ?

Reply:
No just delete the line. I have added all the needed lines bellow it. Please check